tests/CMakeLists.txt

```diff
@@ -7,4 +7,8 @@ foreach (source IN LISTS test-sources)
   add_library(ctre-test-${test} STATIC EXCLUDE_FROM_ALL ${source})
   target_link_libraries(ctre-test-${test} PRIVATE ctre)
   add_dependencies(ctre-test ctre-test-${test})
+
+  if (MSVC)
+    target_compile_options(ctre-test-${test} PRIVATE /utf-8)
```

Comment:
You probably mean `/source-charset:utf-8`

Reply:
`/utf-8` implies `/source-charset:utf-8`

Reply:
It also implies execution-charset:utf-8, which is a much bigger hammer that I don't think you intended on wielding.
In particular, it's not something a library ought to change - nor is useful in the context of the bug you are trying to fix

Reply:
This only sets it for the tests, it doesn't impact users